cinema/migrations/0001_initial.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 5.1.2 on 2024-10-23 00:13
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = []
+
+    operations = [
+        migrations.CreateModel(
+            name="Movie",
+            fields=[
+                (
+                    "id",
+                    models.BigAutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                ("title", models.CharField(max_length=255)),
+                ("description", models.CharField(max_length=255)),
```

Comment:
The 'duration' field is defined as an IntegerField, which is appropriate for storing the duration in minutes as specified in the task description.